mnswaleh/apisendIT
# #161851928 create user endpoint

#### What does this PR do?
enables creating new user

#### What does this PR do?
gets the number of delivered orders created by a specific user

#### Description of Task to be completed?
should have:
1 model method to add user to database
2 POST /users/ endpoint

#### How should this be manually tested?
1 on postman, enter url http://127.0.0.1:5000/api/v1/users/1
2 send post request with required data
3 should return the user details
4 response code is 201
5 run pytest, the test should pass

#### What are the relevant pivotal tracker stories?
#161851928
#### Screenshots (if appropriate)
![screenshot from 2018-11-10 01-35-49](https://user-images.githubusercontent.com/39084014/48292349-05946b00-e48b-11e8-89fc-146c51a22429.png)
